### PR TITLE
Creating progress circle

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #876
[Problem]
- Wrong Example in wrong place
[Solution]
- Creating new examples

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/etc/progress/progress-circle-indeterminate/progress-circle-indeterminate.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../../lib/tau/mobile/js/tau.js"></script>
+	<title>Progress Circle Indeterminate</title>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<h1>
+				Progress Bar Indeterminate
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/etc/progress/progress-circle-indeterminate/progress-circle-indeterminate.html]: Bar -> Circle

File: examples/mobile/UIComponents/components/etc/progress/index.html [hunk 2]
```diff
@@ -22,6 +22,12 @@ <h1>Progress</h1>
 			<li class="ui-li-anchor">
 				<a href="progress-bar-indeterminate/progress-bar-indeterminate.html">Progress bar Indeterminate</a>
 			</li>
+			<li class="ui-li-anchor">
+				<a href="progress-circle/progresscircle.html">Progress Circle</a>
+			</li>
+			<li class="ui-li-anchor">
+				<a href="progress-circle-indeterminate/progress-circle-indeterminate.html">Progress Circle</a>
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/etc/progress/index.html]: add "Indeterminate"

File: examples/mobile/UIComponents/components/etc/progress/progress-circle/progresscircle.html [hunk 3]
```diff
@@ -3,9 +3,9 @@
 
 <head>
 	<meta content="width=device-width, user-scalable=no" name="viewport" />
-	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../css/style.css" rel="stylesheet" />
-	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	<link href="../../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../../lib/tau/mobile/js/tau.js">
 	</script>
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/etc/progress/progress-circle/progresscircle.html]: in line 30, add class "ui-bottom-button" to footer

File: src/js/core/widget/core/progress/type/indeterminatecircle.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+ * limitations under the License.
+ */
+/*jslint nomen: true, plusplus: true */
+/**
+ * #activityCircle Type
+ * activityCircle type support for Progress widget.
+ * @internal
+ * @class ns.widget.core.progress.type.activitycircle
+ * @extends ns.widget.core.progress.type.interface
+ */
```
Comment (TomaszLukawskiSam) [inline src/js/core/widget/core/progress/type/indeterminatecircle.js:25]: add `@author`

File: examples/mobile/UIComponents/components/etc/progress/progress-circle-indeterminate/progress-circle-indeterminate.html (hunk 1 above)
Comment (korneliakobiela) [reply examples/mobile/UIComponents/components/etc/progress/progress-circle-indeterminate/progress-circle-indeterminate.html]: Done

File: examples/mobile/UIComponents/components/etc/progress/index.html (hunk 2 above)
Comment (korneliakobiela) [reply examples/mobile/UIComponents/components/etc/progress/index.html]: Done

File: examples/mobile/UIComponents/components/etc/progress/progress-circle/progresscircle.html (hunk 3 above)
Comment (korneliakobiela) [reply examples/mobile/UIComponents/components/etc/progress/progress-circle/progresscircle.html]: Done

File: examples/mobile/UIComponents/components/etc/progress/index.html [hunk 5]
```diff
@@ -22,6 +22,12 @@ <h1>Progress</h1>
 			<li class="ui-li-anchor">
 				<a href="progress-bar-indeterminate/progress-bar-indeterminate.html">Progress bar Indeterminate</a>
 			</li>
+			<li class="ui-li-anchor">
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/etc/progress/index.html:25]: Is this proper that "Progress" is placed in "etc" ?
Progress has own category number 8 in guideline.

File: package-lock.json [hunk 6]
```diff
@@ -1,9 +1,70 @@
 {
   "name": "tau",
-  "version": "1.0.27",
+  "version": "1.2.1",
```
Comment (TomaszLukawskiSam) [inline package-lock.json]: Do you want to update this file in this PR?

File: src/css/profile/mobile/common/progress.less [hunk 7]
```diff
@@ -209,188 +209,127 @@ tau-progress {
...
-.ui-activity-circle2 {
-	.animation-name(circle2rotate);
-}
+  .ui-indeterminate-circle {
+	box-sizing: border-box;
+	margin: auto;
+	animation: rotating 2s linear infinite;
+  }
+
+  .ui-indeterminate-circle:before {
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: please use `less` feature:
```
.ui-indeterminate {
     &::before {
         ...
     }
     &::after {
         ...
     }
}

```

File: src/css/profile/mobile/common/progress.less [hunk 8]
```diff
@@ -209,188 +209,127 @@ tau-progress {
...
+  
+  .ui-indeterminate-circle:after {
+	content:"";
+	position: relative;
+	left: 50%;
+	border-right-style: solid;
+	border-bottom-style: solid;
+	border-color: var(--primary-color);
+	display: block;
+	border-radius:100% 0;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: space before "100%"

File: src/css/profile/mobile/common/progress.less [hunk 9]
```diff
@@ -209,188 +209,127 @@ tau-progress {
...
+	box-sizing: border-box;
+	margin: auto;
+	animation: rotating 2s linear infinite;
+  }
+
+  .ui-indeterminate-circle:before {
+	content:"";
+	border-top-style: solid;
+	border-left-style: solid;
+	border-color: #06b485;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: please define this color as `less` and `css` variable

File: src/js/core/widget/core/progress/type/indeterminatecircle.js [hunk 10]
```diff
@@ -0,0 +1,100 @@
+/*global define, ns */
+/*
+ * Copyright (c) 2015 Samsung Electronics Co., Ltd
```
Comment (TomaszLukawskiSam) [inline src/js/core/widget/core/progress/type/indeterminatecircle.js]: 2020

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

File: src/css/profile/mobile/common/progress.less [hunk 11]
```diff
@@ -209,188 +209,128 @@ tau-progress {
...
-	100% {
-		stroke-dashoffset: 500px;
+
+  }
+
+  .ui-indeterminate-circle-medium {
+	width: 48 * @px_base;
+	height: 48 * @px_base;
+
+	&:before, &:after {
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/progress.less]: please use `::before` `::after` (CSS3 syntex)

https://developer.mozilla.org/en-US/docs/Web/CSS/::before

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**